AirCasting/Utils/UserState.swift

```diff
@@ -4,5 +4,13 @@
 import Foundation
 
 class UserState: ObservableObject {
+    enum State {
+        case loggingOut
+        case deleting
+        case none
+    }
+    
     @Published var isLoggingOut = false
+    @Published var isShowingLoading = false
+    @Published var currentState: State = .loggingOut
```

Comment:
as I mentioned before I think that this class should only hold a `currentState` property and client code should do whatever with it. Can it be done?

Reply:
As I mentioned before we need `isShowingLoading ` and `currentState` to display LoadingView
![Screenshot 2022-04-21 at 10 50 06](https://user-images.githubusercontent.com/82096200/164417578-d49843e5-8bd1-4067-b060-9e1c1c4419b1.png)

`isLoggingOut ` was removed from that class

Reply:
@SzymonGorowski take a look at how it can be achieved: https://github.com/HabitatMap/AirCastingiOS/pull/691